src/schema.patch

```diff
@@ -76,8 +86,22 @@ index d0d6a3b..8928a6e 100644
          circulating_tokens_supply -> Numeric,
          total_tokens_supply -> Numeric,
          total_lockup_contracts_count -> Int4,
-@@ -107,13 +107,23 @@ table! {
-         lockups_locked_tokens -> Numeric,
+@@ -109,12 +119,13 @@ table! {
+ }
+ 
+ table! {
+     use diesel::sql_types::*;
+     use crate::models::enums::*;
+ 
++    #[allow(non_snake_case)]
+     assets__non_fungible_token_events (emitted_for_receipt_id, emitted_at_block_timestamp, emitted_in_shard_id, emitted_index_of_event_entry_in_shard, emitted_by_contract_account_id, token_id, event_kind, token_old_owner_account_id, token_new_owner_account_id, token_authorized_account_id, token_memo) {
+         emitted_for_receipt_id -> Text,
+         emitted_at_block_timestamp -> Numeric,
+         emitted_in_shard_id -> Numeric,
+         emitted_index_of_event_entry_in_shard -> Int4,
+         emitted_by_contract_account_id -> Text,
+@@ -126,13 +137,23 @@ table! {
+         token_memo -> Text,
```

Comment:
The table (this whole `assets__non_fungible_token_events` block) should not be part of the patch

Reply:
It's not the part of the patch, only `+    #[allow(non_snake_case)]` is